src/server/protocol.ts

```diff
@@ -681,9 +681,17 @@ namespace ts.server.protocol {
 
     export type OrganizeImportsScope = GetCombinedCodeFixScope;
 
+    export const enum OrganizeImportsMode {

```

Comment:
This feels slightly weird - it's like a flags enum, but the values are all strings and aren't directly related.  I guess that's because `sort` and `combine` can't be manipulated independently?  And we gave `All` a super-general name for future proofing reasons?